docs/tutorial/t-enable-encryption.md

```diff
@@ -0,0 +1,99 @@
+# Enable Security in your Kafka deployment 
+
+This is part of the [Charmed Kafka Tutorial](/t/charmed-kafka-tutorial-overview/10571). Please refer to this page for more information and the overview of the content. 
+
+## Transport Layer Security (TLS)
+[TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) is used to encrypt data exchanged between two applications; it secures data transmitted over the network. Typically, enabling TLS within a highly available database, and between a highly available database and client/server applications, requires domain-specific knowledge and a high level of expertise. Fortunately, the domain-specific knowledge has been encoded into Charmed Kafka. This means (re-)configuring TLS on Charmed Kafka is readily available and requires minimal effort on your end.
+
+Again, relations come in handy here as TLS is enabled via relations; i.e. by relating Charmed Kafka to the [TLS Certificates Charm](https://charmhub.io/tls-certificates-operator). The TLS Certificates Charm centralises TLS certificate management in a consistent manner and handles providing, requesting, and renewing TLS certificates.
+
+> *Note: this tutorial uses [self-signed certificates](https://en.wikipedia.org/wiki/Self-signed_certificate); self-signed certificates should not be used in a production cluster.*
+
+### Configure TLS
```

Comment:
**thought/suggestion:** Up to you, but I might make a small note somewhere in this section, that the reason we relate `kafka`/`zookeeper`/`kafka-test-app` all to the **same** `tls-certificates-operator`, is because all certificates are signed by the same rootCA kept on that operator, and as such are trusted by all related applications. Something small to that effect.

Reply:
Maybe also mention that the enabling/disabling of TLS on Kafka side doesn't - I **think** - allow for full availability during the switch-over. I know ZooKeeper does on it's own, but I think together it's not something that can be done on the fly.

@zmraul - Do I remember that correctly?

Reply:
We don't have full availability during switchover. There is a rolling restart where some brokers will have tls ports and others plaintext ones.

Reply:
I'll add a small bit to mention the same CA, but I feel that a deeper discussion (also around switchover and high-availability) is a little beyond the scope of this tutorial